Source/UI/NewUI/BillingForm/View/BillingFormCellTextField.swift

```diff
@@ -38,6 +45,15 @@ final class BillingFormCellTextField: UITableViewCell {
     required init?(coder _: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+  @objc private func hideKeyboard(gestureRecognizer: UITapGestureRecognizer) {
+    let view = gestureRecognizer.view
+    let loc = gestureRecognizer.location(in: view)
+    let subview = view?.hitTest(loc, with: nil)
+    guard !(subview is SecureDisplayView) else { return }
+    UIApplication.shared.sendAction(#selector(UIApplication.resignFirstResponder), to: nil, from: nil, for: nil)
```

Comment:
Given we have this method `hideKeyboard` written twice, what if we do something like:

```swift
extension UITapGestureRecognizer {
    
    static var keyboardDismissTap: UITapGestureRecognizer {
        let gesture = UITapGestureRecognizer(target: self, action: #selector(hideKeyboard(gestureRecognizer:)))
        gesture.cancelsTouchesInView = false
        return gesture
    }
    
    @objc private static func hideKeyboard(gestureRecognizer: UITapGestureRecognizer) {
        let view = gestureRecognizer.view
        let loc = gestureRecognizer.location(in: view)
        let subview = view?.hitTest(loc, with: nil)
        guard !(subview is SecureDisplayView) else { return }
        UIApplication.shared.sendAction(#selector(UIApplication.resignFirstResponder), to: nil, from: nil, for: nil)
    }
    
}

// Use site
addGestureRecognizer(UITapGestureRecognizer.keyboardDismissTap)
```